apps/web-staking/src/app/hooks/useGetRedemptions.ts

```diff
@@ -1,88 +1,118 @@
-import { useCallback, useEffect, useRef, useState } from 'react';
+import { useEffect, useState, useRef } from 'react';
 import { getNetwork } from "@/services/web3.service";
 import { useAccount } from 'wagmi';
 import { getAllRedemptions, OrderedRedemptions, refreshRedemptions } from '@/services/redemptions.service';
-import useSessionStorage from './useSessionStorage';
+
+interface UseGetRedemptionsResult {
+    redemptions: OrderedRedemptions;
+    loadRedemptions: () => void;
+    redemptionsLoading: boolean;
+}
 
 const INITIAL_REDEMPTIONS: OrderedRedemptions = {
     claimable: [],
     open: [],
     closed: [],
 };
 
-const useGetRedemptions = () => {
+const THROTTLE_DELAY = 1000;
+
+const getStorageValue = <T>(key: string, defaultValue: T): T => {
+    if (typeof window === 'undefined') return defaultValue;
+    try {
+        const stored = window.sessionStorage.getItem(key);
+        return stored ? JSON.parse(stored) : defaultValue;
+    } catch {
+        return defaultValue;
+    }
```

Comment:
Is the `window.` required here ?
Just asking for info.

Reply:
No, you are correct, it's not required. I went ahead and removed.